apps/site/src/pages/settings/billing-settings-panel/billing-overview.tsx

```diff
@@ -228,10 +231,13 @@ export const BillingOverviewPanelPage: FunctionComponent = () => {
           />
         )}
       </Box>
-      {userHasStripeSubscription && <UsageLimitSection />}
+      {stripeSubscriptionStatus === "active" ||
+      stripeSubscriptionStatus === "past_due" ? (
+        <UsageLimitSection />
+      ) : null}
```

Comment:
@thehabbos007 @CiaranMn @nonparibus have made a change here to only display the usage limit section when a user has an active stripe subscription, do you agree with this change? Otherwise the section would be available for users with a cancelled stripe subscription, or an incomplete one.

Reply:
We want the `Usage Limit` section to be visible to everybody all of the time, including free users (as they may have expired subscriptions, but active payment methods still attached). And others may want to set usage limits pre-emptively ahead of signing up.

Reply:
Understood, have changed it to always be displayed in https://github.com/blockprotocol/blockprotocol/pull/1068/commits/7671ec70a76e597841c7fa612a008619a7b356d5. @thehabbos007 let me know if you forsee any issues with displaying the input for users without stripe subscriptions, seems to be working for me locally.